### PR TITLE
Make httpbin samples work on OpenShift

### PR DESCRIPTION
**Pull request by jwendell**

### CODE REVIEW

**Review by craigbox — COMMENTED** [review]
LGTM, though I might wait for Eric to weigh in too - assume they continue to work fine on other distros?

File: releasenotes/notes/make-httpbin-work-ocp.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+apiVersion: release-notes/v2
+kind: bug-fix
+area: documentation
+releaseNotes:
+- |
+    **Fixed** `httpbin` samples now work on OpenShift.
```
Comment (dhawton) [inline releasenotes/notes/make-httpbin-work-ocp.yaml:6]:
```suggestion
    **Fixed** `httpbin` sample manifests to deploy correctly on OpenShift.
```

**Comment by jwendell** [comment]
Yeah, I tried on kind as well just to make sure nothing breaks.

**Comment by jwendell** [comment]
According to Slack, Eric's on vacation :/

**Review by dhawton — APPROVED** [review]
Tested locally with kind and validated working.

**Comment by ericvn** [comment]
It looks like this might be causing an istio.io test failure in the authentication/authn-ppolicy tests, specifically the last part of https://istio.io/latest/docs/tasks/security/authentication/authn-policy/#enable-mutual-tls-per-workload. After setting portLevelMtls to DISABLE, The test result has:
```
sleep.legacy to httpbin.bar: 000
command terminated with exit code 56
```
while the page shows:
```
sleep.legacy to httpbin.bar: 200
```
This occcurs for both the gateway api and istio api versions of the test.

**Comment by zirain** [comment]
after this PR, httpbin request python.org always.

**Pull request merged**